ui/api/topics/actions.ts

```diff
@@ -177,6 +186,7 @@ export async function setTopicAsViewed(kafkaId: string, topicId: string) {
       topicId,
       // name is included in the `fields[topics]` param list so we are sure it is present
       topicName: topic.attributes.name!,
+      topicStatus: topic.attributes.status!,
```

Comment:
Just something to be aware of is that the topic status in the overview card will always have the status that it had at the point it was last viewed. Are we OK with that? Having the status be up-to-date would require `getViewedTopics` to be more complex to actually load the topics fresh.

Reply:
Thinking about this again, maybe it would be better to just list the topic name and not the status to avoid it being stale? What do you think @hemahg @shirimordechay ?

Reply:
@MikeEdgar As discussed, I have removed the status field from the table for now.